clms-api/src/main/resources/application.yaml

```diff
@@ -12,11 +12,12 @@ spring:
   cloud:
     aws:
       s3:
-        endpoint: ${SPRING_CLOUD_AWS_S3_ENDPOINT:https://s3.us-east-2.amazonaws.com}
+        endpoint: ${SPRING_CLOUD_AWS_S3_ENDPOINT:http://s3.localhost.localstack.cloud:4566}
         region: ${SPRING_CLOUD_AWS_S3_REGION:us-east-1}
+        path-style-access-enabled: true
```

Comment:
weird this path-style-access-enabled worked ( https://stackoverflow.com/a/77907066 )